### PR TITLE
build: card component

### PR DESCRIPTION
**Pull request by AyaQunoo**

### CODE REVIEW

File: client/src/components/Card/card.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { Card , Rate,Button } from 'antd';
+import {
+  SendOutlined
+} from '@ant-design/icons';
+import './Card.modules.css'
+function CardDiv() {
+  return (
+    <Card style={{ width: 300 }}>
+      <div className='head'>
+        <img src='https://th.bing.com/th/id/OIP.f3DM2upCo-p_NPRwBAwbKQHaHa?pid=ImgDet&rs=1' style={{width:70 , height:70}}/>
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: Antd Image Component

File: client/src/components/Card/card.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Card , Rate,Button } from 'antd';
+import {
+  SendOutlined
+} from '@ant-design/icons';
+import './Card.modules.css'
+function CardDiv() {
+  return (
+    <Card style={{ width: 300 }}>
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: dont use inline style

File: client/src/components/Card/card.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Card , Rate,Button } from 'antd';
+import {
+  SendOutlined
+} from '@ant-design/icons';
+import './Card.modules.css'
+function CardDiv() {
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: CardDiv is not descriptive name for this component

File: client/src/components/Card/card.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import './Card.modules.css'
+function CardDiv() {
+  return (
+    <Card style={{ width: 300 }}>
+      <div className='head'>
+        <img src='https://th.bing.com/th/id/OIP.f3DM2upCo-p_NPRwBAwbKQHaHa?pid=ImgDet&rs=1' style={{width:70 , height:70}}/>
+      <Button type="dashed" icon={<SendOutlined />} shape="round" >
+           Message
+          </Button></div>
+       <h2>Aya Qunoo</h2>
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: this name recived from component props

File: client/src/components/Card/card.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      
+       <div className='bottom'>
+       <Rate disabled defaultValue={2} />
+       <div className='flags'>
+        <img src="https://cdn-icons-png.flaticon.com/128/206/206657.png" alt="" />
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: Use antd Image Component

**Review by devstm — CHANGES_REQUESTED** [review]

File: client/src/components/Card/card.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```
Comment (devstm) [inline client/src/components/Card/card.tsx]: this file name should  be index.tsx

File: client/src/components/Card/Card.modules.css [hunk 7]
```diff
@@ -0,0 +1,28 @@
+.css-dev-only-do-not-override-12jzuas.ant-btn-dashed:hover {
```
Comment (devstm) [inline client/src/components/Card/Card.modules.css]: style.module.css

File: client/src/pages/Community/community.modules.css [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.community {
```
Comment (devstm) [inline client/src/pages/Community/community.modules.css]: style.module.css

**Review by devstm — CHANGES_REQUESTED** [review]

File: client/src/pages/Community/index.tsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Col, Row , Input} from 'antd';
+import { Button, Dropdown,  Space } from 'antd';
```
Comment (abdullah-alsharif) [inline client/src/pages/Community/index.tsx]: merge between line 2 and 3

File: client/src/pages/Community/index.tsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Col, Row , Input} from 'antd';
+import { Button, Dropdown,  Space } from 'antd';
+import { DownOutlined , SearchOutlined } from '@ant-design/icons';
+import type { MenuProps } from 'antd';
+import  CardDiv  from '../../components/Card/card.tsx';
+import './community.modules.css'
+import menuItems from './menuItems';
```
Comment (abdullah-alsharif) [inline client/src/pages/Community/index.tsx]: maybe u can use menuItems as component is better

File: client/src/components/Card/card.tsx [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+      <Button type="dashed" icon={<SendOutlined />} shape="round" >
+           Message
+          </Button></div>
+       <h2>Aya Qunoo</h2>
+      
+       <div className='bottom'>
+       <Rate disabled defaultValue={2} />
+       <div className='flags'>
+        <img src="https://cdn-icons-png.flaticon.com/128/206/206657.png" alt="" />
+       <img src="https://cdn-icons-png.flaticon.com/128/555/555613.png" alt="" /></div>
```
Comment (abdullah-alsharif) [inline client/src/components/Card/card.tsx]: let the img component be changeable (use props for example)

File: client/src/App.tsx [hunk 12]
```diff
@@ -6,7 +6,7 @@ import {
 } from 'react-router-dom';
 
 import { Signup } from './pages';
-
+import Community from './pages/Community';
```
Comment (MohammedMahani) [inline client/src/App.tsx]: we have index.js in (pages) folder > so you should import it from ( ./pages )

**Pull request closed without merging**